identity-apps-core/apps/recovery-portal/src/main/webapp/tenant-resolve.jsp

```diff
@@ -53,7 +54,7 @@
             if (StringUtils.isNotBlank(tenantDomainOfUser)) {
                 tenantForTheming = tenantDomainOfUser;
             }
-            if (StringUtils.equals(spAppName, "My Account") 
+            if (StringUtils.equals(spAppName, "My Account")
```

Comment:
Any framework util to get `"My Account"` rather than hardcoding here?

Reply:
This is a formatting change in this PR. (Removal of trailing space)